utils.js

```diff
@@ -61,12 +63,15 @@ function getBinaryPath () {
   // run async function synchronously because testem doesn't like async configs
   // I'm not proud of this but it works.
   const versionBuffer = childProcess.execSync(`
-    node -e "
+    CHROMIUM_CHANNEL=${REQUESTED_CHANNEL} CHROMIUM_VERSION=${REQUESTED_VERSION} node -e "
```

Comment:
This is the main fix in the PR.  Without setting, non-env shell vars will be lost.  This makes it difficult for users to set the variables externally.

Reply:
Just thinking about this now.  I could quote these variables to be extra safe.  But, hopefully the strings don't have special chars or spaces.  Thoughts?

Reply:
Safe enough for now as is, I think. We can cross that bridge later if we need to